online-shopping-website/frontend/src/Components/Browse/Products/Slideshow.js

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import 'react-slideshow-image/dist/styles.css';
+import { Slide } from "react-slideshow-image";
+import { Link } from "react-router-dom";
+
+//for every promoted product, add it in the slideshow
+const ProductShow = (props) => {
+  console.log(props);
+  return(
+    <Link to={{
+      pathname: `/${props.product.id}/${props.product.name}`,
+      params: { props }
+    }} className="RoutingLink">
+      <div className="each-slide">
+        <div>
+          <div className="CenteringProductDetails">
+            <div className="ProductDetailsShow">
+              <img className="ProductImageShow" src={props.product.picture} alt={props.product.name} />
+              <div>
+                <h3 className="ProductInfoSlide">{props.product.name}</h3>
+                <p className="ProductInfoSlide">Brand: {props.product.brand.name}</p>
+                <p className="ProductInfoSlide">Sold by: {props.product.seller.sellerName}</p>
+              </div>
```

Comment:
```suggestion
              <div className="ProductInfoSlide">
                <h3>{props.product.name}</h3>
                <p>Brand: {props.product.brand.name}</p>
                <p>Sold by: {props.product.seller.sellerName}</p>
              </div>
```
I would suggest this, just for best practices, saving some repeated code, and a small speed increase 😎